pincer/middleware/interaction_create.py

```diff
@@ -125,12 +128,14 @@ async def interaction_response_handler(
     \\*\\*kwargs :
         The arguments to be passed to the command.
     """
-    sig, params = get_signature_and_params(command)
-    if should_pass_ctx(sig, params):
+    # Prevent args from being mutated unexpectedly
+    args = copy(args)
```

Comment:
What is the purpose of `copy(args)` ?
Cannot be `list(args)` ?

Reply:
the intent behind `copy(args)` is more clear